server/modlog/index.ts

```diff
@@ -31,7 +31,8 @@ const PUNISHMENTS = [
 	'UNRANGEBAN', 'TRUSTUSER', 'UNTRUSTUSER', 'BLACKLIST', 'BATTLEBAN', 'UNBATTLEBAN',
 	'NAMEBLACKLIST', 'KICKBATTLE', 'UNTICKETBAN', 'HIDETEXT', 'HIDEALTSTEXT', 'REDIRECT',
 	'NOTE', 'MAFIAHOSTBAN', 'MAFIAUNHOSTBAN', 'GIVEAWAYBAN', 'GIVEAWAYUNBAN',
-	'TOUR BAN', 'TOUR UNBAN', 'UNNAMELOCK',
+	'TOUR BAN', 'TOUR UNBAN', 'UNNAMELOCK', 'FORCEWEEKLOCK', 'FORCELOCK', 'FORCEMONTHLOCK',
+	'FORCERENAME OFFLINE',
```

Comment:
`GLOBAL_PUNISHMENTS` are those punishments that count towards a user's autolocks. All of these forced punishments should probably go in that category, unless there's policy reasoning to convince me otherwise.

Reply:
Thanks for clarifying, I'll move some things around tonight

Reply:
Does this look right?